<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>border_style_dialog</class>
 <widget class="QDialog" name="border_style_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Border Style</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="border_style_group">
     <property name="title">
      <string>Border Style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QLabel" name="border_style_label">
        <property name="text">
         <string>Line Style:</string>
        </property>
        <property name="buddy">
         <cstring>border_style</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="border_color_label">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>border_color</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="border_color">
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="border_style">
        <item>
         <property name="text">
          <string>solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dotted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashdot</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="border_size_label">
        <property name="text">
         <string>Line Width:</string>
        </property>
        <property name="buddy">
         <cstring>border_size</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="ScientificDoubleSpinBox" name="border_size">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000001000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="highlight_style_group">
     <property name="title">
      <string>Selected Mask Highlight</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="highlight_color_label">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>border_color</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="highlight_color">
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="opacity_label">
        <property name="text">
         <string>Opacity</string>
        </property>
        <property name="buddy">
         <cstring>border_size</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ScientificDoubleSpinBox" name="opacity">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>border_color</tabstop>
  <tabstop>border_style</tabstop>
  <tabstop>border_size</tabstop>
  <tabstop>highlight_color</tabstop>
  <tabstop>opacity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>border_style_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>border_style_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
